pip_build.py

```diff
@@ -45,14 +46,37 @@
 ]
 
 
-def build():
+def export_version_string(version, is_nightly=False):
+    """Export Version and Package Name."""
+    if is_nightly:
+        date = datetime.datetime.now()
+        version += f".dev{date.strftime('%Y%m%d%H')}"
+        # Replaces `name="keras"` string in `setup.py` with `keras-nightly`
```

Comment:
Replaces package name and dependencies for the nightly package.

Reply:
Done